### ISSUE TITLE
match sim timing to robot

### ISSUE THREAD

**Issue by Armandpl** [Armandpl/furuta#27]
One discrepancy between the sim and the actual robot is the time at which we get the observation.

In the sim we execute the action in simulation so we get the state + observation + reward at t+1.
On the robot we can't wait for dt before reading the state (or else we don't have time for other computations such as neural net maths). So we execute the action and read the robot state and we get the observation + reward at t. This introduces some delay that we can compensate for using an HistoryWrapper. Unsure if we could do some parallel computing to read the state at t+1? not really bc we need the reward to backprop and stuff?

In any case we can read the state at t in the sim. This should help for sim2real

**Comment by Armandpl** [comment]
To do:

- [ ] return obs at t instead of t+1
- [ ] add eval wrapper + early stopping
- [ ] setup env + wrappers + algo + robot w/ hydra
- [ ] redo the seeding

Then run sweep: 4 diff seeds + historywrapper 1- 4. Does it even converge without the HistoryWrapper
